druid/Dockerfile

```diff
@@ -36,7 +36,11 @@ RUN curl --fail -L "https://repo.stackable.tech/repository/packages/druid/apache
     mvn clean install -Pdist -pl '!extensions-core/druid-ranger-security' -DskipTests -Dmaven.javadoc.skip=true && \
     tar -xzf /stackable/apache-druid-${PRODUCT}-src/distribution/target/apache-druid-${PRODUCT}-bin.tar.gz && \
     mv /stackable/apache-druid-${PRODUCT}-src/apache-druid-${PRODUCT} /stackable/apache-druid-${PRODUCT} && \
-    rm -rf /stackable/apache-druid-${PRODUCT}-src
+    rm -rf /stackable/apache-druid-${PRODUCT}-src && \
```

Comment:
Is this necessary? 

At the copy step we only copy the binary and the bin/ folder. I think the 'src' folder shouldn't end up in the final image.

Reply:
We do this to make the intermediate image smaller due to the size constraints on Github Action runners